### ISSUE TITLE
loan prediction

### ISSUE THREAD

**Issue by Marfein** [Marfein/credit#1]
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.tree import DecisionTreeClassifier
from sklearn.metrics import accuracy_score

data = pd.read_csv('credit_approval.csv')

X = data.drop('approved', axis=1)
y = data['approved']

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

model = DecisionTreeClassifier(random_state=42)
model.fit(X_train, y_train)

y_pred = model.predict(X_test)

accuracy = accuracy_score(y_test, y_pred)
print("Accuracy:", accuracy)

test_data = pd.read_csv('credit_approval_test.csv')

if "approved" in test_data.columns:
    test_data.drop('approved', axis=1, inplace=True)

prediction = model.predict(test_data)

print("Predicted approval:", prediction)

**Comment by Marfein** [comment]
[credit_approval_test.csv](https://github.com/Marfein/credit/files/11379770/credit_approval_test.csv)
